apps/a11y-tests/scripts/start-generate-sarif-reports.js

```diff
@@ -0,0 +1,14 @@
+// @ts-check
+
+const concurrently = require('concurrently');
+
+concurrently(
+  [
+    'cd ../fabric-website-resources && node ../../scripts/just.js webpack-dev-server',
```

Comment:
Will these forward slash paths work on Windows?

Reply:
Yes. Both on Mac and Windows (cmd/PowerShell).